Jenkinsfile_CNP

```diff
@@ -12,7 +12,8 @@ def component = "tribunals-frontend"
 def secrets = [
   'sscs-${env}':
     [
-      secret('idam-sscs-oauth2-client-secret', 'SERVICES_IDAM_SECRET')
+      secret('idam-sscs-oauth2-client-secret', 'SERVICES_IDAM_SECRET'),
+      secret('postcode_lookup_token', 'POSTCODE_LOOKUP_TOKEN')
```

Comment:
don't think this is needed here. Don't think it's needed for the IDAM either, really.

Reply:
are we sure shall I remove this ?

Reply:
try it without it and if fails then put it back, please.

Reply:
Can you tell me how all this works, what configuration is used for what?